Makefile.am

```diff
@@ -16,7 +16,7 @@ SUBDIRS = rust src plugins qa rules doc etc python ebpf \
           $(SURICATA_UPDATE_DIR)
 DIST_SUBDIRS = $(SUBDIRS) examples/lib/simple examples/lib/custom
 
-CLEANFILES = stamp-h[0-9]*
+CLEANFILES = stamp-h[0-9]* compile_commands.json
```

Comment:
Can we put this is distclean? I like to keep my `compile_commands.json` over make clean.  If its not created by `make` it should probably not be removed by `make clean`.